.changeset/busy-mangos-ask.md

```diff
@@ -0,0 +1,17 @@
+---
+"@fluidframework/container-definitions": minor
```

Comment:
Since the `Container` API surface affected by this, I think it makes sense to call out that package here. Also that will cause the CHANGELOG.md file for it to be updated with this notes when we release.
```suggestion
"@fluidframework/container-definitions": minor
"@fluidframework/container-loader": minor
```

Reply:
Container class is not exported so I didn't think we should point it out, right?

Reply:
Ah, that's a great point, yeah.